ui/src/app/pages/profile/profile-component.tsx

```diff
@@ -429,21 +435,23 @@ export const ProfileComponent = fp.flow(
             </div>
           </div>
           <div style={{width: '20rem', marginRight: '4rem'}}>
-            <div style={styles.title}>Free credits balance
+            <div style={{marginLeft: '1rem'}}>
```

Comment:
There was a visual alignment bug which became more obvious with this change.  It's probably easier to view this file with whitespace diffs off.